symplyphysics/conditions/electricity/maxwell_equations/faradays_law_of_induction.py

```diff
@@ -0,0 +1,64 @@
+from sympy import Eq, Derivative
+from sympy.core import Equality
+from symplyphysics import (units, Quantity, Symbol, validate_input)
+from symplyphysics.core.fields.operators import curl_operator as rotor
+from symplyphysics.core.fields.vector_field import VectorField
+from symplyphysics.core.dimensions import assert_equivalent_dimension
+
+## Description
+## Faraday's law of induction states that a change in magnetic induction generates a vortex electric field.
+
+## Law is: rot(E(x, y, z, t)) = -d(B(x, y, z, t)) / dt, where
+## E - the vector of electric intensity,
+## B - the vector of magnetic induction,
+## x, y, z - coordinates in a rectangular coordinate system,
+## t - tiime,
+## rotor - rotor of the vector of magnetic induction,
+## d / dt - time derivative.
+
+x_coor = Symbol('x_coor', units.length)
+y_coor = Symbol('y_coor', units.length)
+z_coor = Symbol('z_coor', units.length)
+time = Symbol('time', units.time)
+
+
+def faradays_law_of_induction(electric_intensity_: VectorField,
+                                            magnetic_induction_: VectorField,
+                                            time_: Quantity) -> tuple[Equality, Equality, Equality]:
+    rotor_electric_intensity = rotor(electric_intensity_)
+
+    x = rotor_electric_intensity.coordinate_system.coord_system.base_scalars()[0]
+    y = rotor_electric_intensity.coordinate_system.coord_system.base_scalars()[1]
+    z = rotor_electric_intensity.coordinate_system.coord_system.base_scalars()[2]
+    dict_xyz = {
+        x: x_coor,
+        y: y_coor,
+        z: z_coor,
+        time_: time,
+    }
+
+    law_x = Eq(rotor_electric_intensity.apply_to_basis().components[0].subs(dict_xyz), -Derivative(magnetic_induction_.apply_to_basis().components[0].subs(dict_xyz), (time, 1)))
+    law_y = Eq(rotor_electric_intensity.apply_to_basis().components[1].subs(dict_xyz), -Derivative(magnetic_induction_.apply_to_basis().components[1].subs(dict_xyz), (time, 1)))
+    law_z = Eq(rotor_electric_intensity.apply_to_basis().components[2].subs(dict_xyz), -Derivative(magnetic_induction_.apply_to_basis().components[2].subs(dict_xyz), (time, 1)))
+    return law_x, law_y, law_z
```

Comment:
I think we only need one Equality. Of vectors or vector fields.

Reply:
> I think we only need one Equality. Of vectors or vector fields.

Yes, I tried. But such equality leads to an error. The code is below.
```python
def faradays_law_of_induction(electric_intensity_: VectorField, 
                              magnetic_induction_: VectorField) -> tuple[Equality, Equality, Equality]:
    rotor_electric_intensity = rotor(electric_intensity_)
    magnetic_induction_space = magnetic_induction_.apply_to_basis()
    magnetic_induction_time_derivative = [-Derivative(c, time) for c in magnetic_induction_space.components]
    magnetic_induction_time_derivative_vector = Vector(magnetic_induction_time_derivative, magnetic_induction_.coordinate_system)
 
    return Eq(rotor_electric_intensity, VectorField.from_vector(magnetic_induction_time_derivative_vector))
```

Error:
SympifyError: <symplyphysics.core.fields.vector_field.VectorField object at 0x0000020A4EAB9C50>

Reply:
But I can implement it through matrices.
```python
def faradays_law_of_induction(electric_intensity_: VectorField, 
                              magnetic_induction_: VectorField) -> tuple[Equality, Equality, Equality]:
    rotor_electric_intensity = rotor(electric_intensity_)
    rotor_electric_intensity_space = rotor_electric_intensity.apply_to_basis()

    magnetic_induction_space = magnetic_induction_.apply_to_basis()
    magnetic_induction_time_derivative = [-Derivative(c, time) for c in magnetic_induction_space.components]
    magnetic_induction_time_derivative_vector = Vector(magnetic_induction_time_derivative, magnetic_induction_.coordinate_system)

    return Eq(Matrix(rotor_electric_intensity_space.components), Matrix(magnetic_induction_time_derivative))
```

Reply:
You can apply vector field to basis. This way you'll be able to sympify it.